### DISCUSSION TITLE
can't add new song please answer

### DISCUSSION THREAD

**Discussion by loogchin** [KadeArchive/Kade-Engine#1676] [Q&A] [answered]
can't add new song pls answer

I tried adding custom songs several times but I just can't.
Err code:[ERROR] Could not find a Sound asset with an id of 'song:assest/song/your song/inst.ogg'.
ERROR] Could not find a Sound asset with an id of 'song:assest/song/your song/Voices.ogg'.
Even if I use the .ogg file :(

I solved the problem by renaming the song to 'null', but I would like to use my custom song name.

**Comment by raf13lol** [comment, marked as answer]
No spaces put song/your-song/Inst.ogg  song/your-song/Voices.ogg
same thing with data 
name in game is fine just edit the storymenustate.hx file

**Reply by loogchin** [reply]
https://i.imgur.com/RfWQrhK.png

**Reply by raf13lol** [reply]
why are you using 1.5.4?

**Reply by raf13lol** [reply]
anyways is that the source code or what?

**Reply by raf13lol** [reply]
try making the 'T' into a 't'

**Reply by loogchin** [reply]
it soure code

**Reply by loogchin** [reply]
i amnot add new song
Even if I give another version, I can't add songs.

**Reply by loogchin** [reply]
> try making the 'T' into a 't'

thank you
